core-bundle/contao/library/Contao/Controller.php

```diff
@@ -1439,20 +1439,6 @@ public static function addEnclosuresToTemplate($objTemplate, $arrItem, $strKey='
 		$objTemplate->enclosure = $arrEnclosures;
 	}
 
-	/**
-	 * Set the static URL constants
-	 */
-	public static function setStaticUrls()
```

Comment:
Removing the constants is not a problem, because they have been deprecated every since Contao 4.0:

https://github.com/contao/contao/blob/f28fa5e71d37959b027820ad266a832fa28e6c19/DEPRECATED.md?plain=1#L63-L78

However, removing the static method is a BC break, as the method is not deprecated. @contao/developers Would you prefer to keep the method as a no-op and flag it for removal in Contao 6?

Reply:
We're allowed to break, no? Or deprecate it in 4.13 now?

Reply:
+1, add a deprecation for the next 4.13 release and then remove 👍

Reply:
Agree, remove in 5.0 and add a deprecation in 4.13.

Reply:
Deprecation added in #5194.